.gitignore

```diff
@@ -3,6 +3,6 @@
 *.iml
 .env
 .DS_Store
-venv/
+.venv/
 .pytest_cache/
 **__pycache__/
```

Comment:
The pattern `**__pycache__/` is missing a leading slash to indicate it's a directory. Consider changing it to `**/__pycache__/` for consistency with other directory patterns.